### PR TITLE
Apply mocpy 0.5.* API changes

### PR DESCRIPTION
**Pull request by bmatthieu3**
This is not directly related to https://github.com/astropy/astroquery/issues/1342 but the API of MOCPy changed for the MOC.write method since v0.5.* and this caused some remote tests to fail.

I also deployed a new version of mocpy which should correct https://github.com/astropy/astroquery/issues/1342.

### CODE REVIEW

**Comment by keflavich** [comment]
@bsipocz it looks like this would help with some of our issues, and travis passed.  The appveyor failure looks like a compiler failure in a dependency.  Good to merge?

**Comment by bmatthieu3** [comment]
The problem with appveyor relies on a new dependency of MOCPy : Spherical geometry https://github.com/spacetelescope/spherical_geometry

There is no binary wheels for this package so the sources are downloaded instead and the package is installed from the sources.
I do not know why but for the architecture of windows used and python 3.6 it fails during the installation. Works with python2.7.
Did you have similar problems with other dependencies whose installation from sources failed ?

File: astroquery/cds/core.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def _args_to_payload(self, **kwargs):
             region = kwargs['region']
             if isinstance(region, MOC):
                 self.path_moc_file = os.path.join(os.getcwd(), 'moc.fits')
-                region.write(format='fits', write_to_file=True, path=self.path_moc_file)
+                region.write(self.path_moc_file, format="fits")
```
Comment (bsipocz) [inline astroquery/cds/core.py]: would this also work with older mocpy? If not, then the minimum version number of 0.5 should be listed in the docs (docs/index.rst).

**Comment by bsipocz** [comment]
@bmatthieu3 - In case the dependencies are not picked up by pip or conda, then we usually do the workaround to list the manually for the relevant section. `spherical-geometry` is available on the astropy conda channel, so you can choose whether to list it in `CONDA_DEPENDENCIES` or it's also fine to list it for `PIP_DEPENDENCIES`

**Comment by bmatthieu3** [comment]
I removed spherical-geom from mocpy's dependencies as there was a compiling issue (spherical geom has C extensions) on the [windows py3 job from appveyor](https://ci.appveyor.com/project/Astropy/astroquery/builds/21932596/job/kvyy308mu1an6eb7). The tests are passing now.

**Pull request merged**